pkg/local_instance_manager.go

```diff
@@ -174,6 +174,35 @@ func (im *InstanceManager) Create(instances []Instance) ([]Instance, error) {
 	return created, nil
 }
 
+func (im *InstanceManager) Import(instances []Instance) ([]Instance, error) {
+	imported := []Instance{}
+	log.Info(InstancesMsg(instances, "importing"))
+
+	for _, i := range instances {
+		if !i.local.IsCreated() {
+			err := i.local.Import()
+			if err != nil {
+				return nil, err
+			}
+			imported = append(imported, i)
+		}
+	}
+
+	restarted := []Instance{}
+	for _, i := range imported {
```

Comment:
it would be nice to tell the user why we are restarting here

Reply:
I have sth like this here - https://github.com/wttech/aemc/blob/main/pkg/local_instance_manager.go#L203

Reply:
I mean some msg like `%s > imported but already running - restarting to apply configuration`